lib/ex_unit/lib/ex_unit/manifest_formatter.ex

```diff
@@ -0,0 +1,64 @@
+defmodule ExUnit.ManifestFormatter do
```

Comment:
Let's merge this with the runner stats formatter. What do you think?

Reply:
I can merge it, but I couldn't come up with a good name for the merged formatter.  Also, tracking aggregate stats and individual test statuses are kinda distinct responsibilities.

Regardless, I'll take a stab at merging them in the next couple days, and will leave the name as `ExUnit.RunnerStats` unless someone suggests a better name for the merged module.

Reply:
Sounds good to go with RunnerStats for now as well.